pages/pipelines/conditionals.md.erb

```diff
@@ -0,0 +1,119 @@
+# Conditionals
+
+- the `if` parameter provides the ability to conditionally run steps
+- using c-like expressions, boolean conditions are defined to restrict the running of steps to specific circumstances
+- can be used in all YAML-based pipelines
+
+<%= toc %>
+
+## `if` Syntax
+
+- use the `if` param to conditionally run a step
+- the below example will only run the `tests` step if the build message doesn't contain the string "skip me"
+
+```
+steps:
+	- command: ./scripts/tests.sh
+	  label: tests
+	  - if:
+	  	build.message != "skip me"
+```
+
+- use it in any step type
+- any of the supported expressions and parameters
+- cannot be used at the same time as the `branch` parameter
+
+## Supported Expressions
+
+ - Comparators: == != =~ !~
+ - Logical operators: || &&
+ - Integers 12345
+ - Strings 'foobar' or "foobar"
+ - Booleans true false
+ - Parenthesis to control order of evaluation ( )
+ - Regular expressions /^v1\.0/
+ - Function calls foo("bar")
```

Comment:
I don't think we'll have any functions built-in for launch, so not sure if we should mention this in our docs.

Reply:
Oop, I removed it from the example code but forgot it for this bit 😄